packages/twenty-server/src/modules/contact-creation-manager/jobs/create-company-and-contact.job.ts

```diff
@@ -30,6 +31,7 @@ export class CreateCompanyAndContactJob {
         displayName: contact.displayName,
       })),
       workspaceId,
+      FieldActorSource.EMAIL,
```

Comment:
This one is also emit from CalendarSaveEventsService so it won't always be EMAIL